app/views/events/show.slim

```diff
@@ -19,12 +19,12 @@
 
     -else
       .text-center
-        - if @event.attending?(current_user.id)
+        - if !@event.attending?(current_user.id)
           = link_to 'Attend', attend_event_path(@event), method: :post, class: 'btn btn-success buttons-config'
 
         - else
           = link_to 'Cancel', cancel_attend_event_path(@event), method: :delete, class: 'btn btn-danger buttons-config'
-        - if @event.owner == current_user.id
+        - if @event.owner_id == current_user.id
```

Comment:
Don't you have an Ability class already? This check belongs in there, not on the view. We should only have something like `can? :manage, @event` here